style.css

```diff
@@ -1,3 +1,3 @@
-.bg-color{
-    background-color: purple;
-}
\ No newline at end of file
+.bg-color {
+  background-color: purple;
+}
```

Comment:
- [ ] Kindly add the link to the CSS file in your `index.html` code to see the styles in the browser. 👇🏽 
![image](https://user-images.githubusercontent.com/58642949/191385718-5d340d6f-1972-4c31-ba96-d3317426ec8b.png)